_layouts/default.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{ site.lang | default: "en-US" }}">
+<html lang="{{ page.lang | site.lang | default: "en-US" }}">
```

Comment:
I think this needs to be

```suggestion
<html lang="{{ page.lang | default: site.lang | default: "en-US" }}">
```